L/Lua/build_tarballs.jl

```diff
@@ -31,7 +31,7 @@ fi
 # XXX: Work around Lua apparently not understanding its own Windows setup
 if [[ ${target} == *-mingw* ]]; then
     TO_BIN="lua.exe luac.exe"
-    TO_LIB="lua53.dll"
+    TO_LIB="lua54.dll"
```

Comment:
I wonder what happens if we call this `liblua.dll`? :slightly_smiling_face:

Reply:
That's a good question. My only concern would be for downstream libraries that make naming assumptions on Windows for this library and wouldn't know to look for `liblua.dll`.

Reply:
That's a good point :disappointed:

Reply:
Should probably leave as-is for now, at least.